app/scripts/components/common/chart/utils.ts

```diff
@@ -1,5 +1,6 @@
 import { timeFormat, timeParse } from 'd3';
 import * as d3ScaleChromatic from 'd3-scale-chromatic';
+import { NumberLiteralTypeAnnotation } from '@babel/types';
```

Comment:
This is an unused dependency which I thought is part of the lint rule we have. I thought GitHub action will catch this but it doesn't seem like it. Am I understanding how GitHub Action works now? @danielfdsilva ?

Reply:
It should be. I also noticed that rule not throwing an error lately. Perhaps was disabled by mistake. I'll have a look